<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartTool</class>
 <widget class="QMainWindow" name="ChartTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>778</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift Light SemiCondensed</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ChartTool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>441</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Bahnschrift Light Condensed</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Chart Count</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="txtBrowserShowChart">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>161</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Bahnschrift Light Condensed</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>File to Open</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="txtBrowserOpen">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtOpenFileInput">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnValidPush">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>441</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Bahnschrift Light Condensed</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Now Record</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="txtBrowserRecordChart">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>13</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCloseWindow">
       <property name="font">
        <font>
         <family>Bahnschrift Light Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>161</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Bahnschrift Light Condensed</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Want Append</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txtWantAppend">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAppend">
       <property name="font">
        <font>
         <family>Bahnschrift Light Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Append</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMainWindow">
    <property name="title">
     <string>MainWindow</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuMainWindow"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>edtOpenFileInput</tabstop>
  <tabstop>btnValidPush</tabstop>
  <tabstop>txtWantAppend</tabstop>
  <tabstop>btnAppend</tabstop>
  <tabstop>txtBrowserOpen</tabstop>
  <tabstop>txtBrowserRecordChart</tabstop>
  <tabstop>txtBrowserShowChart</tabstop>
  <tabstop>btnCloseWindow</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCloseWindow</sender>
   <signal>clicked()</signal>
   <receiver>ChartTool</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>689</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
